content/en/apps/guides/database/querying_apdex_telemetry.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Querying Apdex Telemetry Data"
+linkTitle: "Querying Apdex Telemetry Data"
+weight:
+description: >
+  How to use SQL queries to view Apdex score in RDBMS
+relatedContent: >
+  apps/guides/database/rdbms-from-windows
+  apps/guides/database/rdbms-from-mac
```

Comment:
and maybe this page https://docs.communityhealthtoolkit.org/apps/guides/database/#postgresql